Server/src/controllers/admin.controller.js

```diff
@@ -20,20 +20,19 @@ class AdminController {
         });
       }
       if (findUser[0].isadmin === 'true') {
-        return res.status(403).json({
-          status: 403,
-          error: 'Admin not allowed to be a mentor!'
-        });
+        return res.status(403).json({ status: 403, error: 'Admin not allowed to be a mentor!'});
+      }
+      if(findUser[0].ismentor === 'true'){
+        return res.status(400).json({ status: 400, error: 'Already you are a Mentor!'});
```

Comment:
I think the most appropriate status code here is a `409` and not `400`. Also, since this operation is done by the admin, the message `Already you are a Mentor!` is not the most appropriate.